src/diffusers/pipelines/fabric/pipeline_fabric.py

```diff
@@ -0,0 +1,727 @@
+        feedback. The feedback can be given as a list of liked and disliked images.
+
+        Args:
+            prompt (`str` or `List[str]`, *optional*):
+                The prompt or prompts to guide the image generation. If not defined, one has to pass `prompt_embeds`.
+                instead.
+            negative_prompt (`str` or `List[str]`, *optional*):
+                The prompt or prompts not to guide the image generation. If not defined, one has to pass
+                `negative_prompt_embeds` instead. Ignored when not using guidance (i.e., ignored if `guidance_scale` is
+                less than `1`).
+            liked (`List[Image.Image]` or `List[str]`, *optional*):
+                Liked enables feedback through images, encourages images with liked features.
+            disliked (`List[Image.Image]` or `List[str]`, *optional*):
+                Disliked enables feedback through images, discourages images with disliked features.
+            generator (`torch.Generator` or `List[torch.Generator]` or `int`, *optional*):
+                One or a list of [torch generator(s)](https://pytorch.org/docs/stable/generated/torch.Generator.html),
+                can be int. to make generation deterministic.
+            height (`int`, *optional*, defaults to 512):
+                height of the generated image
+            width (`int`, *optional*, defaults to 512):
+                width of the generated image
+            num_images (`int`, *optional*, defaults to 1):
+                The number of images to generate per prompt.
+            guidance_scale (`float`, *optional*, defaults to 7.5):
+                Guidance scale as defined in [Classifier-Free Diffusion Guidance](https://arxiv.org/abs/2207.12598).
+                `guidance_scale` is defined as `w` of equation 2. of [Imagen
+                Paper](https://arxiv.org/pdf/2205.11487.pdf). Guidance scale is enabled by setting `guidance_scale >
+                1`. Higher guidance scale encourages to generate images that are closely linked to the text `prompt`,
+                usually at the expense of lower image quality.
+            num_inference_steps (`int`, *optional*, defaults to 50):
+                The number of denoising steps. More denoising steps usually lead to a higher quality image at the
+                expense of slower inference.
+            output_type (`str`, *optional*, defaults to "pil"):
+                defines the output type of generated image supports "np","pil"
+
+        Examples:
+
+        Returns:
+            [`~pipelines.fabric.FabricPipelineOutput`] or `tuple`: When returning a tuple, the first element is a list
+            with the generated images, and the second element is a list of `bool`s denoting whether the corresponding
+            generated image likely represents "not-safe-for-work" (nsfw) content, according to the `safety_checker`.
```

Comment:
```suggestion
            [`~pipelines.fabric.FabricPipelineOutput`] or `tuple`: 
                If `return_dict` is `True`, [`~pipelines.fabric.FabricPipelineOutput`] is returned,
                otherwise a `tuple` is returned where the first element is a list with the generated images and the
                second element is a list of `bool`s indicating whether the corresponding generated image contains
                "not-safe-for-work" (nsfw) content.
```